modules/server_new/src/main/scala/observe/server/InstrumentSystem.scala

```diff
@@ -39,17 +39,17 @@ trait InstrumentSystem[F[_]] extends System[F] {
 object InstrumentSystem {
   val ObserveOperationsTimeout = 1.minute
 
-  final case class StopObserveCmd[F[_]](self: Boolean => F[Unit])
-  final case class AbortObserveCmd[F[_]](self: F[Unit])
-  final case class PauseObserveCmd[F[_]](self: Boolean => F[Unit])
+  final case class StopObserveCmd[+F[_]](self: Boolean => F[Unit])
```

Comment:
This avoids the warning  "type test cannot be checked at runtime" in match statements.